apptentive/src/main/java/com/apptentive/android/sdk/conversation/ConversationManager.java

```diff
@@ -173,6 +176,7 @@ public boolean loadActiveConversation(Context context) {
 
 				activeConversation.startListeningForChanges();
 				activeConversation.scheduleSaveConversationData();
+				activeConversationLoaded = true;
```

Comment:
Whitespace? Maybe not worth worrying about now though.

Reply:
I think, we should not worry about it now